OpenAPI/kiota/toc.yml

```diff
@@ -77,5 +77,7 @@ items:
   items:
   - name: Add an API to search
     href: add-api.md
+  - name: Use Kiota with Dependency Injection in .NET
+    href: tutorials/dotnet-dependencyinjection.md
```

Comment:
you might want to add an additional dash between dependency and injection here.

Reply:
Will do